src/rgw/rgw_lua_background.cc

```diff
@@ -159,7 +169,7 @@ void Background::run() {
           ldpp_dout(dpp, 1) << "Lua ERROR: " << err << dendl;
           failed = true;
         }
-      } catch (const std::exception& e) {
+      } catch (const std::runtime_error& e) {
```

Comment:
what's the motivation for only catching `runtime_error`? won't this cause us to crash on others?

Reply:
will change here, as well as other places in the code (data and request contexts) in a separate PR.